src/opendr/planning/end_to_end_planning/e2e_planning_learner.py

```diff
@@ -21,31 +21,37 @@
 from stable_baselines3 import PPO
 from stable_baselines3.common.monitor import Monitor
 from stable_baselines3.common.vec_env import DummyVecEnv
-from stable_baselines3.common.results_plotter import load_results, ts2xy
+from stable_baselines3.common.callbacks import CheckpointCallback
 
 from opendr.engine.learners import LearnerRL
 from opendr.engine.constants import OPENDR_SERVER_URL
 
-__all__ = ["rospy", ]
+# __all__ = ["rospy", ]
```

Comment:
if indeed unnecessary, then I think we can remove it altogether not just comment it (likewise in line 15)

Reply:
removed